src/main/java/graphql/schema/idl/SchemaGeneratorPostProcessing.java

```diff
@@ -1,13 +1,19 @@
 package graphql.schema.idl;
 
+import graphql.DeprecatedAt;
 import graphql.PublicSpi;
 import graphql.schema.GraphQLSchema;
 
 /**
  * These are called by the {@link SchemaGenerator} after a valid schema has been built
  * and they can then adjust it accordingly with some sort of post processing.
+ *
+ * @deprecated This mechanism can be achieved in a better way via {@link graphql.schema.SchemaTransformer}
+ * after the schema is built
  */
 @PublicSpi
+@Deprecated
+@DeprecatedAt(value = "2012-10-29")
```

Comment:
Nit: should be `@DeprecatedAt("2022-10-29")`

Reply:
fixed